packages/components/inline-message/src/inline-message.ts

```diff
@@ -22,50 +22,50 @@ export type InlineMessageVariant = 'info' | 'success' | 'warning' | 'danger';
 export type SlDismissEvent = CustomEvent<void>;
 
 /**
- * An inline message component for displaying additional information/errors.
+ * An inline message component for displaying additional information.
  *
- * @slot default - slot for the main information of the inline-message
- * @slot title - title content for the inline message
- * @slot details - slot for more details of the inline-message like list of errors
```

Comment:
Can you please also update the example of `inline-message` in the `code` tab on the **website** since there is no `details` slot anymore? :)

<img width="976" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/ba1ec6fb-4f18-4299-abc6-e8e03998134a">